### PR TITLE
[UPDATE] Bump version Fulcrum and small fix

### PR DESCRIPTION
**Pull request by twofaktor**
#### What

Update Fulcrum guide to newly released v1.9.7 and a small fix of a missing variable environment addition on a command

### Why

Keeping up with the latest developments and improvements (see [release notes](https://github.com/cculianu/Fulcrum/releases/tag/v1.9.7)).

#### How

- Updated variable environment and outputs
- Fixed a command

#### Scope

- [ ] significant change to core configuration
- [X] independent bonus guide
- [ ] simple bug fix

#### Test & maintenance

Tested for myself. Working fine ✅

### CODE REVIEW

**Review by blckbx — APPROVED** [review]
LGTM

File: guide/bonus/bitcoin/fulcrum.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ We have our Bitcoin Core configuration file set up, and now we can move to next
   > You can also use the latest release version (`$LATEST_VERSION`). However, please be aware that newer release versions might not have been thoroughly tested with the rest of the RaspiBolt configuration.
 
   ```sh
-  $ VERSION="1.9.6"
+  $ VERSION="1.9.7"
   $ cd /tmp
   $ wget https://github.com/cculianu/Fulcrum/releases/download/v$VERSION/Fulcrum-$VERSION-arm64-linux.tar.gz
   $ wget https://github.com/cculianu/Fulcrum/releases/download/v$VERSION/Fulcrum-$VERSION-sha256sums.txt.asc
```
Comment (kristapsk) [inline guide/bonus/bitcoin/fulcrum.md:193]: There's 404. It seems to be renamed from `Fulcrum-$VERSION-sha256sums.txt.asc` to `Fulcrum-$VERSION-shasums.txt.asc`. Should be changed here and below.
```suggestion
  $ wget https://github.com/cculianu/Fulcrum/releases/download/v$VERSION/Fulcrum-$VERSION-shasums.txt.asc
```

**Review by kristapsk — CHANGES_REQUESTED** [review]

File: guide/bonus/bitcoin/fulcrum.md (hunk 1 above)
Comment (twofaktor) [reply guide/bonus/bitcoin/fulcrum.md:193]: Ok, it was modified 45 min ago by Fulcrum dev 😅
imagen

I will modify that

**Review by twofaktor — COMMENTED** [review]
@kristapsk could you re-review this and approve it?

**Comment by kristapsk** [comment]
@twofaktor Yes, but later.

File: guide/bonus/bitcoin/fulcrum.md [hunk 2]
```diff
@@ -203,9 +203,9 @@ We have our Bitcoin Core configuration file set up, and now we can move to next
 * Verify the signature of the text file containing the checksums for the application
 
   ```sh
-  $ gpg --verify Fulcrum-$VERSION-sha256sums.txt.asc
-  > gpg: assuming signed data in 'Fulcrum-1.9.6-sha256sums.txt'
-  > gpg: Signature made Sat 11 Nov 2023 05:52:23 CET
+  $ gpg --verify Fulcrum-$VERSION-shasums.txt.asc
+  > gpg: assuming signed data in 'Fulcrum-1.9.7-sha256sums.txt'
+  > gpg: Signature made Wed 22 Nov 2023 21:29:49 CET
```
Comment (kristapsk) [inline guide/bonus/bitcoin/fulcrum.md:208]: It shows Nov 24 signature date for me.

**Review by kristapsk — APPROVED** [review]

**Pull request merged**